pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -918,6 +920,35 @@ def test_timedelta_hash_equality(self):
         ns_td = Timedelta(1, "ns")
         assert hash(ns_td) != hash(ns_td.to_pytimedelta())
 
+    @pytest.mark.xfail(
+        reason="pd.Timedelta violates the Python hash invariant (GH#44504).",
+        raises=AssertionError,
+        strict=True,
```

Comment:
Our pytest configuration is strict by default so this isn't needed

Reply:
Will fix

Reply:
Explicit strictness requirement is now dropped.